nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/test/java/org/nuxeo/elasticsearch/TestESHistoryProvider.java

```diff
@@ -191,152 +371,33 @@ protected void createTestEntries() throws Exception {
         createdEntry.setDocPath(doc.getPathAsString());
         createdEntry.setRepositoryId("test");
 
-        List<LogEntry> entries = new ArrayList<>();
-        entries.add(createdEntry);
-        Framework.getService(AuditLogger.class).addLogEntries(entries);
+        auditLogger.addLogEntries(List.of(createdEntry));
 
         LogEntryGen.flushAndSync();
-        List<LogEntry> logs = reader.getLogEntriesFor(doc.getId(), doc.getRepositoryName());
+        transactionalFeature.nextTransaction();
+        List<LogEntry> logs = auditReader.getLogEntriesFor(doc.getId(), doc.getRepositoryName());
         logs.forEach(entry -> log.trace("LogEntry: {}", entry));
     }
 
-    @Test
-    public void testDocumentHistoryPageProvider() throws Exception {
```

Comment:
Why did you move the whole `testDocumentHistoryPageProvider` method? It makes the diff unreadable.

Reply:
i did it because, i went to start by test methods and then let the helpers methods at the end to be focused only on test when we read the class.

Reply:
i rollback this modification to make it at it was before and to simplifie your review